src/smart-components/group/groups.js

```diff
@@ -125,6 +145,14 @@ Groups.propTypes = {
   isLoading: PropTypes.bool,
   searchFilter: PropTypes.string,
   fetchGroups: PropTypes.func.isRequired,
+  selectedRows: arrayOf(
```

Comment:
`selectedRows` and `setSelectedRows` comes from state, so no need to add them to propTypes at all.

Reply:
done! :slightly_smiling_face: